### PR TITLE
ignore the clear.md shit idk whats goin on with dat, anyway play docs

### PR DESCRIPTION
**Pull request by zAG6z**
as always as always change what u need

### CODE REVIEW

File: docs/commands/clear.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ The minimum amount of messages you can delete at once is 2. The maximum you can
 * Usage: `{prefix}clear <messagesToDelete>`
 
 ## Aliases
+
```
Comment (xf8b) [inline docs/commands/clear.md:21]: can you remove the newline i think its why im blocked from merging

**Review by xf8b — CHANGES_REQUESTED** [review]

File: docs/commands/clear.md (hunk 1 above)
Comment (xf8b) [reply docs/commands/clear.md:21]:
```suggestion
```

**Comment by zAG6z** [comment]
so did u do it or what

**Comment by zAG6z** [comment]
cus u merged???/

**Comment by xf8b** [comment]
yea i did it

**Comment by zAG6z** [comment]
alr

**Pull request merged**